content/practices/team-shared-history.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2018-06-28T00:00:00-05:00
+title: Team Shared History
+draft: false
+stages: ['delivery']
+---
+
+## What is it?
+
+A method of helping a team start to become cohesive, develop a shared sense of history, and a shared vision of the future. This simple exercise starts with a facilitator organizing the participants in order of their hire date with the company. Then a timeline is drawn and each team member marks their hire date on the timeline. Next, team members note on the timeline interesting things which may have occurred near their time of hire. Finally, the team member who has the most seniority will "tell the story" of the team's history and perhaps embellish with additional context around what has been going on around the organization.
+
+Outcomes
+- A better understanding between team members of their own place in the organization
+- Establish empathy and context for future conversations between team members
+- Build a sense of community for newer team members
+
+## Why Use Team Shared History?
+
+- New team members are not sure of their place, responsibilities, rights, and history of their org.
+- Team members with more history may assume that newer members know what is expected
+
+## Related Practices
+
+
+## Who do you need?
```

Comment:
you are listing roles in a company. For this practice, I would simplify this to be "everybody on your team". As this list could be exhaustive. The intent of the practice seems to want to include everybody you can that is involved in the team.